entrypoint.sh

```diff
@@ -16,5 +16,6 @@ if [ ! -f "./.env" ]; then
 
   echo "Secrets successfully retrieved."
 fi
-
+echo "Checking if primary admin emails exist"
+npm run add-admin-emails
```

Comment:
I think the docker should run the script now? I can't really check because my docker doesn't work :(

Reply:
It works on my local

Reply:
yup I think it works ty!